llvm/lib/Target/SBF/BTFDebug.cpp

```diff
@@ -26,7 +26,6 @@
 #include "llvm/Target/TargetLoweringObjectFile.h"
 
 using namespace llvm;
-using namespace BTFX;
```

Comment:
This removes a local change we need. The BTFX namespace is needed to wrap our copy when the LLVM back-end libraries for BPF and SBF are linked in the Rust compiler.